test/steps/kogitoapp.go

```diff
@@ -44,10 +44,13 @@ func registerKogitoAppSteps(s *godog.Suite, data *Data) {
 	s.Step(`^Deploy spring boot example service "([^"]*)"$`, data.deploySpringBootExampleService)
 	s.Step(`^Deploy spring boot example service "([^"]*)" with persistence$`, data.deploySpringBootExampleServiceWithPersistence)
 	s.Step(`^Create service "([^"]*)"$`, data.createService)
+	s.Step(`^Create service "([^"]*)" with runtime requests "([^"]*)" and limits "([^"]*)"$`, data.createServiceWithRuntimeRequestsAndLimits)
+	s.Step(`^Deploy quarkus example service "([^"]*)" with build requests "([^"]*)" and limits "([^"]*)"$$`, data.deployQuarkusExampleServiceWithRuntimeRequestsAndLimits)
```

Comment:
minor - Can you please move this step to other "Deploy quarkus example service" steps to keep them together?

Reply:
Well spotted! I will